workflows/sf-tractomics.nf

```diff
@@ -102,18 +103,22 @@ workflow SF_TRACTOMICS {
         ENSEMBLE_TRACKING(ch_tractogram_math_input)
         ch_input_tracking_qc = ENSEMBLE_TRACKING.out.trk
     }
-    else {
+    else if (params.run_local_tracking || params.run_pft_tracking) {
         ch_input_tracking_qc = TRACTOFLOW.out.pft_tractogram
             .mix(TRACTOFLOW.out.local_tractogram)
     }
-    QC_ENSEMBLE(ch_input_tracking_qc
-            .join(TRACTOFLOW.out.wm_mask)
-            .join(TRACTOFLOW.out.gm_mask))
-    ch_sub_multiqc_files = ch_sub_multiqc_files.mix(QC_ENSEMBLE.out.mqc)
-    ch_global_multiqc_files = ch_global_multiqc_files.mix(
-        QC_ENSEMBLE.out.dice.map { _meta, dice_file -> dice_file } )
-    ch_global_multiqc_files = ch_global_multiqc_files.mix(
-        QC_ENSEMBLE.out.sc.map { _meta, sc_file -> sc_file } )
+
+    if (params.run_local_tracking || params.run_pft_tracking) {
```

Comment:
You can also create an empty ch_input_tracking_qc and remove the if condition here.